### PR TITLE
Add dstu2 for encounter, condition

### PR DESCRIPTION
**Pull request by amlanima**

### CODE REVIEW

File: lib/resources.rb [hunk 1]
```diff
@@ -1205,6 +1205,66 @@ def get_resource(key)
...
+                                "clinicalStatus"=>"confirmed",
+                                "onsetDateTime"=>"2007"}}]
+    }
+
+    DSTU2_ENCOUNTER_BUNDLE ||={
+        "resourceType"=>"Bundle",
+        "id"=>"40be3b40-6945-4db9-b993-2f67a44c77a6",
+        "type"=>"searchset",
+        "base"=>"https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total"=>10,
```
Comment (naveenaluru) [inline lib/resources.rb]: should be just 2 to match number of entries in bundle

Comment (jfrhong) [inline lib/resources.rb]: nit, this should be 2

**Comment by jfrhong** [comment]
+1

**Comment by naveenaluru** [comment]
:+1:

**Comment by MaxPhilips** [comment]
+1 after comments addressed

File: lib/resources.rb [hunk 2]
```diff
@@ -1205,6 +1205,66 @@ def get_resource(key)
...
+        "total"=>2,
+        "link"=>[{"relation"=>"self", "url"=>"https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Encounter?patient=1316020"}],
+        "entry"=>[{"resource"=>
+                       {"resourceType"=>"Encounter",
+                        "id"=>"2787906",
+                        "status"=>"in-progress",
+                        "patient"=>{"reference"=>"Patient/1316020"},
+                        "participant"=>[{"type"=>[{"coding"=>[{"system"=>"http://hl7.org/fhir/v3/ParticipationType",
+                                                               "code"=>"ATND", "display"=>"attender"}], "text"=>"attender"}],
+                                         "individual"=>{"display"=>"Timilsina, Manisha"}}], "period"=>{"start"=>"2015-01-05T22:50:48.000Z"},
```
Comment (dennispatterson) [inline lib/resources.rb]: We probably don't want to use Manisha, but rather a fake name.

File: content/dstu2/condition.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Condition | FHIR API
```
Comment (dennispatterson) [inline content/dstu2/condition.md]: The other pages have "FHIR DSTU 2 API" in the title

File: lib/resources.rb [hunk 4]
```diff
@@ -1205,6 +1205,66 @@ def get_resource(key)
...
+
+    DSTU2_ENCOUNTER_BUNDLE ||={
+        "resourceType"=>"Bundle",
+        "id"=>"40be3b40-6945-4db9-b993-2f67a44c77a6",
+        "type"=>"searchset",
+        "base"=>"https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total"=>2,
+        "link"=>[{"relation"=>"self", "url"=>"https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Encounter?patient=1316020"}],
+        "entry"=>[{"resource"=>
+                       {"resourceType"=>"Encounter",
```
Comment (dennispatterson) [inline lib/resources.rb]: Refer directly to the DSTU2_ENCOUNTER rather than duplicating the data.

**Comment by dennispatterson** [comment]
Applied PR comments in Amlanima's absence.

**Comment by dennispatterson** [comment]
Updated the parameter types to be reference and added a link to the spec.

**Comment by dennispatterson** [comment]
Formatted the search parameter table markdown.

**Comment by MaxPhilips** [comment]
+1 for updates

**Comment by jfrhong** [comment]
+1

**Comment by naveenaluru** [comment]
:+1:

**Comment by dennispatterson** [comment]
Squashed and rebased .  Still need another +1.

@daliboz 
@kpshek

**Comment by daliboz** [comment]
+1

**Pull request merged**